ckan_cloud_operator/providers/db/proxy/azuresql/manager.py

```diff
@@ -102,6 +102,11 @@ def _apply_deployment(db_prefix=None):
             'replicas': 1,
             'revisionHistoryLimit': 10,
             'strategy': {'type': 'RollingUpdate', },
+            'selector': {
+                'matchLabels': {
+                    'app': 'provider-db-proxy-azuresql'
```

Comment:
we have a function to generate the labels - why not use it?
(i.e. `_get_resource_labels(for_deployment=True, suffix=db_prefix or ''),`)

Reply:
done

Reply:
where?